chess/__init__.py

```diff
@@ -626,14 +626,15 @@ def __init__(self, from_square, to_square, promotion=None, drop=None):
         self.promotion = promotion
         self.drop = drop
 
-    def uci(self):
+    def pacn(self):
         """
-        Gets an UCI string for the move.
+        Gets an PACN string for the move.
```

Comment:
That was a mistake on my part, PACN stands for Pure Algebraic Coordinate Notation
I got it from [The Chess Programming Wiki](https://chessprogramming.wikispaces.com/Algebraic+Chess+Notation#Pure%20coordinate%20notation)